libkbfs/key_bundle_v3.go

```diff
@@ -109,28 +109,34 @@ func (udkimV3 UserDeviceKeyInfoMapV3) removeDevicesNotIn(
 	updatedUserKeys UserDevicePublicKeys) ServerHalfRemovalInfo {
 	removalInfo := make(ServerHalfRemovalInfo)
 	for uid, dkim := range udkimV3 {
+		userRemoved := false
 		deviceServerHalfIDs := make(deviceServerHalfRemovalInfo)
+		if _, hasUpdatedUser := updatedUserKeys[uid]; hasUpdatedUser {
```

Comment:
might as well save the result of `updatedUserKeys[uid]` in a var since it's used later in a loop.